packages/eservice-event-consumer/src/handlers/messageHandlerV1.ts

```diff
@@ -158,20 +118,18 @@ export async function handleMessageV1(
     .exhaustive();
 }
 
-export const fromEserviceEventToEserviceEntity = (
+export const fromEserviceEventV1ToEserviceEntity = (
   eServiceId: string,
   descriptorsData: EServiceDescriptorV1[],
   streamId: string,
   version: number
-): EserviceEntity => {
-  return {
-    eservice_id: eServiceId,
-    descriptors: descriptorsData.map((descriptor) => ({
-      descriptor_id: descriptor.id,
-      state: descriptor.state as unknown as string, // TODO: to fix
-    })),
-
-    event_stream_id: streamId,
-    event_version_id: version,
-  };
-};
+): EserviceEntity => ({
+  eservice_id: eServiceId,
+  descriptors: descriptorsData.map((descriptor) => ({
+    descriptor_id: descriptor.id,
+    state: descriptor.state as unknown as string, // TODO: to fix
```

Comment:
To fix for next time? Or it can be fixed in this PR?

Reply:
Next time. We need to handle this not only here but on DB and also on BL for signals-service.